cli/cmd/compliance.go

```diff
@@ -211,63 +214,70 @@ func complianceReportRecommendationsTable(recommendations []api.ComplianceRecomm
 }
 
 func buildComplianceReportTable(detailsTable, summaryTable, recommendationsTable [][]string, filteredOutput string) string {
-	mainReport := &strings.Builder{}
-	mainReport.WriteString(
-		renderCustomTable(
-			[]string{
-				"Compliance Report Details",
-				"Non-Compliant Recommendations",
-			},
-			[][]string{[]string{
-				renderCustomTable([]string{}, detailsTable,
-					tableFunc(func(t *tablewriter.Table) {
-						t.SetBorder(false)
-						t.SetColumnSeparator("")
-						t.SetAlignment(tablewriter.ALIGN_LEFT)
-					}),
-				),
-				renderCustomTable([]string{"Severity", "Count"}, summaryTable,
-					tableFunc(func(t *tablewriter.Table) {
-						t.SetBorder(false)
-						t.SetColumnSeparator(" ")
-					}),
-				),
-			}},
-			tableFunc(func(t *tablewriter.Table) {
-				t.SetBorder(false)
-				t.SetAutoWrapText(false)
-				t.SetColumnSeparator(" ")
-			}),
-		),
-	)
-
-	if compCmdState.Details || complianceFiltersEnabled() {
+	if compCmdState.Csv {
```

Comment:
The check for CSV should live along side the json flag check and not in the buildComplianceReportTable function.

It'd be great if there was symmetry across the alternative outputs.

`			if cli.JSONOutput() {
				return cli.OutputJSON(report)
			}

			if compCmdState.Csv {
				return cli.OutputCSV(report)
			}
`

Reply:
@dmurray-lacework The reason I did not do this is because the CSV output should be the same data we feed through the tables (which has had some further processing than the raw response).  That's not the case with JSON output where we drop the raw body.  I felt it was better to not have duplicate code to process the response body (which would be required in the suggestion above).  

Thoughts on this?